modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentConfiguration.java

```diff
@@ -97,23 +97,23 @@
 
     /** */
     @Parameter(names = {"-cs", "--client-certificate"},
```

Comment:
Also, please, add client certificate, client password for web console backend.

Reply:
Done